### ISSUE TITLE
Files marked for download are not remembered across connection loss

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/android#222]
1. Multiselect some files and then download them
2. Go into airplane mode, have bad connection or smth similar
3. Observe the error (The error should be more specific about the connection loss btw)
4. Reconnect after some time
5. Go back to the app, and the files are not marked for marked for download anymore, nor automatically downloaded.

**Comment by tobiasKaminsky** [comment]
True, we have no download queue like we have for upload.
Mainly because uploads should not start immediately, but instead have some restrictions (like wifi, charging or doze mode), whereas a download should start immediately as the user wants to have the file *now*.

While I see the benefit of having this, implementing this will be currently too much work for too little benefit, I think.
@mario @AndyScherzinger ?

**Comment by AndyScherzinger** [comment]
I agree on the capacity issue we have right now as in "not enough time/people to also tackle this", else I think it is a valid enhancement which might also come into play for "available offline" by simply having an upload and download queue where the client makes sure up/downloads happen over time when connection losses or whatever happens. 
It is not a really visible feature in the end but will improve the overall experience a lot since "the client takes care" so the users don't have to check if something went wrong and would have to remember the files they wanted to download and manually redo the action.

I'll label it enhancement, since it is not really a bug but a "missing" behavior.
